src/App.js

```diff
@@ -1,39 +1,81 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+// import React from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
+import React, { useState, useEffect } from 'react'; 
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
+const [homeScore, setHomeScore] = useState(0);
+const [awayScore, setAwayScore] = useState(0);
+const [seconds, setSeconds] = useState(0);
+const [minutes, setMinutes] = useState(0);
+const [isOn, setIsOn] = useState(false);
+
+const touchdownHandler = (setScore) => {
+  return () => setScore(score => score + 7);
+}
+
+function toggle(){
+  setIsOn(!isOn);
+}
+function reset(){
+  setSeconds(0);
+  setMinutes(0);
+  setIsOn(false);
+}
+
```

Comment:
writing these functions up here is a great way to add clarity to your code. Good work!